<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>241</width>
     <height>158</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="tanggalKirimLabel">
      <property name="text">
       <string>Tanggal Kirim</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="tanggalKirimDateEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPengirimLabel">
      <property name="text">
       <string>Nama Pengirim</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="linePengirim"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emailPengirimLabel">
      <property name="text">
       <string>Email Pengirim</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="isiPesanLabel">
      <property name="text">
       <string>Isi Pesan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineIsi"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="iDMemberLabel">
      <property name="text">
       <string>ID Member</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="iDMemberLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emailPengirimLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="simpanBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ubahBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cetakBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CETAK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>491</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>TANGGAL KIRIM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NAMA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISI PESAN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID MEMBER</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="cariEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
